Sources/BraintreePayPal/BTPayPalApprovalURLParser.swift

```diff
@@ -32,6 +32,17 @@ struct BTPayPalApprovalURLParser {
                 return ecToken
             }
 
+            return nil
+
+        case .payPalApp(let url):
```

Comment:
Maybe `BTPayPalApprovalURLParser` instead have an `ecToken` & `baToken` property and the logic on which to use lives in `BTPayPalClient`?

Reply:
We can for sure, I pushed a commit here (https://github.com/braintree/braintree_ios/pull/1274/commits/d676ee2462d0ab064c19a077824c2689d78275e0) so we can decide if we like this approach. It made sense to also set the URL in the init (like we do with redirectType) since the redirectType doesn't matter for extracting these properties. Curious to hear what you think of this approach!